modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+=== Footnotes 1.0.0
+
+The {productname} 6.2 release includes the new **Footnotes** premium plugin.
+
+The Footnotes plugin allows the user to add footnotes to {productname} documents.
+
+For information on the **Footnotes** plugin, see: xref:footnotes.adoc[Footnotes].
+
+=== Merge Tags 1.0.0
+
+The {productname} 6.2 release includes the new **Merge Tags** premium plugin.
+
+The Merge Tags plugin allows the user to insert merge tags (also known as a personalization tokens, or a mail merge fields) into {productname} documents.
+
+For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tags].
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.2.
+
+=== Enhanced Media Embed 3.0.4
+
+The {productname} 6.2 release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.0.4 fixes the following bug:
+
+Previously, embedded audio objects became invisible when a left or right alignment was applied.
+
+With this update all audio objects are set to `min-width: 100px` and now continue to show in {productname} documents when aligned left or right.
+
+For information on the **Enhanced Media Embed** plugin, see: xref:introduction-to-mediaembed.adoc[Enhanced Media Embed].
+
+=== Permanent Pen 2.0.2
+
+The {productname} 6.2 release includes an accompanying release of the **Permanent Pen** premium plugin.
+
+Permanent Pen 2.0.2 fixes the following bug:
+
+Previously, if Permanent Pen was switched off (by, for example, clicking the Permanent Pen button in the toolbar so the button toggles from highlighted to not-highlighed) and the next user-action was to press the *Enter* or *Return* key, Permanent Pen did not turn off as expected.
+
+This update improves Permanent Pen’s activate and deactivate mechanism. When the plugin is switched off and the next user-action is to press either *Enter* or *Return*, Permanent Pen stays switched off.
+
+For information on the **Permanent Pen** plugin, see: xref:permanentpen.adoc[Permanent Pen].
+
+=== PowerPaste 6.1.1
+
+The {productname} 6.2 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+PowerPaste 6.1.1 fixes the following bug:
+
+The `wordimport.js` resource file was difficult to bundle when using Webpack 5.
+
+// #TINY-8791
+
+// Bumped internal dependency of wimp to fix webpack 5 bundling issues
+
+For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
+
+=== Real-Time Collaboration 2.1.1
+
+The {productname} 6.2 release includes an accompanying release of the **Real-Time Collaboration** premium plugin.
+
+Real-Time Collaboration 2.1.1 fixes the following bug:
+
+The Real_Time Collaboration plugin validates `that` parameters, used to set the token provider among other settings, as valid Promises.
```

Comment:
```suggestion
The Real-Time Collaboration plugin validates that parameters, used to set the token provider among other settings, are valid link:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise[Promises].
```
Typo, links and minor edits as this didn't really make sense. I'm not sure it still does tbh, but it's more correct.

Reply:
> Typo, links and minor edits as this didn't really make sense.

I’ve committed your edit. That said:

> I'm not sure it still does tbh, but it's more correct.

Does this make more sense?

_The Real-Time Collaboration plugin checks that parameters used to set the token provider, among other settings, are valid link:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise[Promises]._